src/index.js

```diff
@@ -14,29 +14,23 @@ import { NotFoundPage } from './pages/NotFound';
 
 import { LoginPage } from './pages/Login';
 import { config } from './services/okta/oktaConfig';
-import reducer from './state/reducers';
-import { createStore, applyMiddleware, compose } from 'redux';
-import { Provider } from 'react-redux';
-import thunk from 'redux-thunk';
 import Landing from './pages/Landing/Landing';
 
 // Seller Imports
 import SellerProfile from './pages/SellerProfilePage';
-import Inventory from './pages/InventoryPage';
-import CurrentInventory from './pages/CurrentInventoryPage';
+import { Inventory } from './pages/InventoryPage';
+import { CurrentInventory } from './pages/CurrentInventoryPage';
 import { ProductPage } from './pages/ProductPage';
-import { TestItemImageUpload } from './components/common';
-
-const store = createStore(reducer, compose(applyMiddleware(thunk)));
+import { AuthProvier } from './contexts/auth/AuthProvider';
 
 ReactDOM.render(
-  <Provider store={store}>
+  <AuthProvier>
```

Comment:
is this a misspell? should be AuthProvider?

Reply:
I accidentally checked out this branch off of the remove redux branch - this was addressed on that pull request. We decided to actually remove the provider all together and create a useOktaId custom hook instead.

Reply:
The Security provider that okta provides already handled much of what I wanted to do with this provider - it was a premature decision to create this provider without reading okta docs 😁